components/post_view/reaction_list/reaction_list.jsx

```diff
@@ -97,15 +92,15 @@ export default class ReactionListView extends React.PureComponent {
     }
 
     render() {
-        if (!this.props.post.has_reactions || (this.props.reactions && this.props.reactions.length === 0)) {
+        if (!(this.props.post.has_reactions && this.props.reactions)) {
```

Comment:
0/5, but I think this is more clearly expressed as:
```suggestion
        if (!this.props.post.has_reactions || !this.props.reactions) {
```

Reply:
I had that first and then changed it.  0/5 as well, will change it.